vizro-core/docs/pages/user-guides/extensions.md

```diff
@@ -91,7 +91,13 @@ enabling an interaction between data points in a scatter plot and the content of
         Vizro().build(dashboard).run()
         ```
 
+## CSS customizations
+
+Although Vizro is opinionated about visual formatting and certain elements like the navigation and control layout are
+fixed, you can still customize various settings, including background colors, fonts, and other styles, through CSS
+overrides.
```

Comment:
```suggestion
Although Vizro is opinionated about visual formatting and certain elements like the navigation and control layout,
you can still customize various settings, including background colors, fonts, and other styles, through CSS
overrides.
```

Reply:
I used the same version as in customise-css.md now :) That one was already signed off by Jo 👍